### PR TITLE
Refactor kthxbye python code

### LINKED ISSUE #1083: [refacto] data indexation
Avec l'utilisation de datasets npm versionnés et/ou du JSON, on devrait passer pour simplifier à une indexation des datas ES en JS.

Toutes les sources sont un simple dump des fichiers JSON dans `dataset`, sauf les fiches ministère travail qui ont un traitement particulier

Il faudra faire attention aux changements de slug et l'intégration dans la CI

### PR DESCRIPTION
**Pull request by lionelB**
- replace python code by js
- update Dockerfile and ci

fix #1083

### CODE REVIEW

**Comment by lionelB** [comment]
don't worry, I forgot to update the readme

File: .gitlab-ci/variables.yml [hunk 1]
```diff
@@ -11,8 +11,7 @@ variables:
   GIT_STRATEGY: fetch
   HASH_SIZE: 7
   IMAGE_INFRA_BASE_NAME: infra/images-docker
+  PYTHON_VERSION: 3.7.2-alpine
```
Comment (douglasduteil) [inline .gitlab-ci/variables.yml]: Not on alphabetical order :(

File: packages/code-du-travail-data/package.json [hunk 2]
```diff
@@ -13,5 +21,26 @@
   "bugs": {
     "url": "https://github.com/SocialGouv/code-du-travail-numerique/issues"
   },
-  "homepage": "https://github.com/SocialGouv/code-du-travail-numerique/tree/master/packages/code-du-travail-data#readme"
+  "homepage": "https://github.com/SocialGouv/code-du-travail-numerique/tree/master/packages/code-du-travail-data#readme",
+  "dependencies": {
+    "@elastic/elasticsearch": "^7.3.0",
+    "esm": "^3.2.25",
```
Comment (douglasduteil) [inline packages/code-du-travail-data/package.json:28]: Esm is a devdep no ?

**Review by douglasduteil — CHANGES_REQUESTED** [review]

File: packages/code-du-travail-data/package.json (hunk 2 above)
Comment (lionelB) [reply packages/code-du-travail-data/package.json:28]: it is used by check-slug script too

**Comment by lionelB** [comment]
@douglasduteil updated !

**Review by revolunet — APPROVED** [review]
🎉

File: .gitlab-ci/variables.yml (hunk 1 above)
Comment (lionelB) [reply .gitlab-ci/variables.yml]: done !

File: packages/code-du-travail-data/package.json (hunk 2 above)
Comment (lionelB) [reply packages/code-du-travail-data/package.json:28]: and in dump script too

**Review by douglasduteil — APPROVED** [review]

**Pull request merged**